### ISSUE TITLE
Flex-wrap issue in mini-match-results

### ISSUE THREAD

**Issue by 2Pacalypse-** [ShieldBattery/ShieldBattery#885]
![image](https://user-images.githubusercontent.com/4757944/205045595-505bde97-c3f3-4d47-b5d5-41e5e54b24cc.png)

**Comment by williamhammond** [comment]
Is the wrap issue here that one of the players isn't showing?

**Comment by 2Pacalypse-** [comment]
No, there's only one player against 2 computers. So the left column should show the "Top" header and the player name, and the right column should show the "Bottom" header and the two computers. In the screenshot, the "Bottom" header is shown in the left column.

FWIW, this might be easier to achieve with CSS Grid, but I'm not sure.
